ccd-definition/CaseEventToComplexTypes/draftSDO-localAuthorityCustom.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
Is there a reason this file has a different name format from the other files? The others are `draftSDO-blahDirections` and  `draftSDO-blahDirectionsCustom`.

Reply:
Especially as the ID seems to follow the convention

Reply:
hmm this wasn't the issue I thought it was when we spoke the other day... I shall update